### DISCUSSION TITLE
[How-to] - How to convert string to pendulum

### DISCUSSION THREAD

**Discussion by BarryAljawari** [sdispater/pendulum#651] [Q&A] [answered]
Hello beautiful people!

I have for hours and hours trying to figure out how to use pendulum in a case which is super hard. I have a text that returns 

`01-06-08-45`

which means:

Closing in : 1 day, 6 hours, 8 minutes and 45 seconds.

and I want to be able to convert this into pendulum.format/parse (so I have something like 2022-09-01 12:00:00)
I have currently done something like this:
```
t = '01-06-08-45'
test = pendulum.duration(days=float(t.split('-')[0]), hours=float(t.split('-')[1]), minutes=float(t.split('-')[2]), seconds=float(t.split('-')[3]))
```
which returns `1 day 6 hours 8 minutes 45 seconds` but my question is how can I convert this into a format of e.g. `2022-09-01 12:00:00` instead? Is that possible using Pendulum?

**Comment by NickFabry** [comment, marked as answer]
Hi @BarryAljawari - I think you need to make your question more clear. You are creating a pendulum.Duration object (something that represents a length of time), yet you want to display a format that only makes sense for a pendulum.Datetime object (something that designates a particular moment in time).

If you want to explore different ways of turning a duration into a string, have a look at the docs about them: https://pendulum.eustace.io/docs/#properties-and-duration-methods . If you want to convert the Duration into a DateTime, you have to realize that doesn't make sense without more context. That's like saying, I'm going to wait for 4 hours and 52 minutes; express that as a calendar year, month, date, hour and minute.
